x/ecocredit/msgs.go

```diff
@@ -125,7 +125,7 @@ func (m *MsgCreateBatch) ValidateBasic() error {
 			}
 
 			if !retiredAmount.IsZero() {
-				if err = ValidateLocation(iss.RetirementLocation); err != nil {
+				if err = ValidateJurisdiction(iss.RetirementLocation); err != nil {
```

Comment:
Looks like issuance still needs to be updated for `MsgCreateBatch`.

Reply:
i dont think its really necessary to update these, it's pretty much just waiting to be deleted at this point. all the messages have already been migrated to `core/` 🤷🏻

Reply:
Yup, that makes sense.